cmd/scheduler/app/options/options.go

```diff
@@ -88,6 +97,8 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 	fs.DurationVar(&o.SchedulerEstimatorTimeout.Duration, "scheduler-estimator-timeout", 3*time.Second, "Specifies the timeout period of calling the scheduler estimator service.")
 	fs.IntVar(&o.SchedulerEstimatorPort, "scheduler-estimator-port", defaultEstimatorPort, "The secure port on which to connect the accurate scheduler estimator.")
 	fs.BoolVar(&o.EnableEmptyWorkloadPropagation, "enable-empty-workload-propagation", false, "Enable workload with replicas 0 to be propagated to member clusters.")
+	fs.StringSliceVar(&o.Plugins, "plugins", []string{"*"},
+		fmt.Sprintf("A list of plugins to enable. '*' enables all on-by-default plugins, 'foo' enables the plugin named 'foo', '*,-foo' disables the plugin named 'foo'.  \nAll build-in plugins: %s.", strings.Join(frameworkplugins.NewInTreeRegistry().FactoryNames(), ",")))
```

Comment:
I see you added a `\n`, but why with two blank spaces. What's the flag look like, I mean the output of `karmada-scheduler --help`.

Reply:
![image](https://user-images.githubusercontent.com/17962021/177968743-9473fbb7-8663-464b-8198-c93fbfd7eba7.png)

Reply:
yes, but no big deal.